orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy

```diff
@@ -86,7 +86,7 @@ class CreateBakeTask implements RetryableTask {
           stage.context.user = user
         }
       }
-    } catch (RetrofitError e) {
+    } catch (SpinnakerServerException e) {
       // ignore exception, we will just use the owner passed to us
       if (!e.message.contains("404")) {
```

Comment:
Something doesn't seem right about this...like, why not check the response code...and why not re-throw the exception if it's not ignored?  Let's close our eyes in this PR.

Reply:
Ok. Lets not cover this in this PR